### DISCUSSION TITLE
How can i add Chips to Menu

### DISCUSSION THREAD

**Discussion by Lunatix01** [mantinedev/mantine#1675] [Q&A] [answered]
**Hey there**
Menu Items are buttons, but I'm trying to change it to something like a div and add chips or a checkbox. 
I used component prop doesn't work. so is it possible to chips or checkboxes to Menu?

_thanks for answering_

**Comment by rtivital** [comment, marked as answer]
You can provide custom menu item component to control root element: `<Menu.Item component={YourComponent} />`. I would not recommend that because in most cases it will be an accessibility violation.

**Reply by Lunatix01** [reply]
thanks it fixed my problem. but any other way to avoid accessibility violation like any alternative component? for example if I want to create a filter button that displays a menu and select which to be filtered

**Reply by rtivital** [reply]
Use Popover component instead of Menu

**Reply by Lunatix01** [reply]
> Use Popover component instead of Menu

Thanks <3

**Reply by mattaningram** [reply]
Would be really useful if we had a `<Menu.Checkbox>` type component the way Radix does so we don't need to lose all the benefits of the Menu component when we just need a filter dropdown.
